types/station.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	"strconv"
+
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/twpayne/go-geom"
+	"github.com/twpayne/go-geom/encoding/ewkb"
+	_ "github.com/twpayne/pgx-geom"
+	"golang.org/x/net/html"
+)
+
+type Station struct {
+	WebsiteID pgtype.Text `json:"websiteID" db:"website_id"`
+	PublicID  pgtype.Text `json:"publicID" db:"public_id"`
```

Comment:
Just looking at these names, I'm not entirely sure what they depict, do both come from the nlwkn website?

Reply:
They both come from the [nlwkn website]. The `PublicID` represents the ID listed directly on the website. However, the website itself uses a different ID when navigating, which is hidden in the view displayed.

[nlwkn website]: https://www.grundwasserstandonline.nlwkn.niedersachsen.de/Messwerte